ui/src/app.js

```diff
@@ -725,15 +733,20 @@ function touchEvent (element, callback) {
 
     e.stopPropagation()
     e.preventDefault()
-  })
+  }
+
+  element.addEventListener('touchstart', handler)
+  element.addEventListener('mousedown', handler)
 }
 
 function touchImmediateEvent (element, callback) {
```

Comment:
Is this used anywhere?

Reply:
`TouchImmediateEvent`? Yep, used by directly by two buttons and indirectly by every button created with `setupImmediateButton`